data-structures/queue/README.md

```diff
@@ -0,0 +1,30 @@
+# Data Structures - ABT - Queue
+
+**Author** : Jay Adams
+**Version**: 0.1.0
+
+## Overview
+Working with Queues
+
+## Getting Started
+Specification
+Create a new branch in your data-structures-and-algorithms repository called stack-queue. 
+
+Create a directories: queue, with all of your standard module configuration for each directory
+
+__init__.py, README.md, etc.
+Create two files in each called node.py and  queue.py respectively
```

Comment:
The user does not need to create these files, thus this should not be in the README.